content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -95,10 +95,10 @@ So, this change is coming. It’s going to cause issues for some, but it isn’t
 catastrophic, and generally it’s a good thing. Depending on how you interact
 with Kubernetes, this could mean nothing to you, or it could mean a bit of work.
 In the long run, it’s going to make things easier. If this is still confusing
-for you, that’s okay -- there’s a lot going on here, Kubernetes has a lot of
+for you, that’s okay&mdash;there’s a lot going on here; Kubernetes has a lot of
 moving parts, and nobody is an expert in 100% of it. We encourage any and all
 questions regardless of experience level or complexity! Our goal is to make sure
-everyone is educated as much as possible on the upcoming changes. `<3` We hope
```

Comment:
> `<3` 

The authors intended to communicate a :heart: here - I'd like to leave that in somehow.

Reply:
@sftim how about putting it at the end? e.g.:

> We hope this has answered most of your questions and soothed some anxieties! `<3`

Reply:
Also we could just use the emoji itself:
> We hope this has answered most of your questions and soothed some anxieties! ❤️

Reply:
@sftim WDYT?

Reply:
I'd be okay with the heart 👍

Reply:
@mrbobbytables Done. PTAL.